salt/journal/config/etc-nginx-sites-available-journal.conf

```diff
@@ -35,22 +35,18 @@ server {
     {% endif %}
 }{% endfor %}
 
+map $http_x_forwarded_host $robots_disallow {
+    hostnames;
 
-# TODO: simplify to use only X-Forwarded-Host after move to Fastly
-# right now, displays the allowing robots.txt when at
-# least one of the two headers correspond to the canonical host
-map "$http_host:$http_x_forwarded_host" $robots_disallow {
     default "/";
-    "~^elifesciences\.org:.*$" "";
-    "~^.*:elifesciences\.org$" "";
+    {{ pillar.elife.domain }} "";
 }
 
 map $http_x_forwarded_host $http_x_forwarded_host_filtered {
+    hostnames;
+
     default "";
-    "~^elifesciences\.org$" $http_x_forwarded_host;
-    "~^(?<subdomain>.*)\.elifesciences\.org$" $http_x_forwarded_host;
-    # Fastly testing domain owned by eLife
-    "~^elifejournal\.org$" $http_x_forwarded_host;
+    .{{ pillar.elife.domain }} $http_x_forwarded_host;
```

Comment:
Not tested, but should work?

Reply:
Should be testable in Vagrant with `curl -v -H "Host: ..." 127.0.0.1/ping`

Reply:
Confirmed as working.